### PR TITLE
DOCS-6483 SAP 5.0 Connector Docs

### PR DESCRIPTION
**Pull request by valkyrie69**

### CODE REVIEW

**Review by sorinmare — APPROVED** [review]

File: modules/ROOT/pages/sap/sap-connector-reference-43.adoc [hunk 1]
```diff
@@ -0,0 +1,802 @@
...
+* SAP:RETRY_EXHAUSTED 
+* SAP:INVALID_HOST_ERROR 
+
+
+== Sources
+
+[[DocumentSource]]
+=== Document Source
+`<sap:document-source>`
+
```
Comment (laszloczol) [inline modules/ROOT/pages/sap/sap-connector-reference-43.adoc:594]: Could you add here 'Source that listens for incoming IDoc documents'

File: modules/ROOT/pages/sap/sap-connector-reference-43.adoc [hunk 2]
```diff
@@ -0,0 +1,802 @@
...
+=== For Configurations
+* <<inbound-config>> 
+
+
+
+[[FunctionSource]]
+=== Function Source
+`<sap:function-source>`
+
```
Comment (laszloczol) [inline modules/ROOT/pages/sap/sap-connector-reference-43.adoc:638]: Could you add here: 'Source that listens for BAPI functions'

File: modules/ROOT/pages/sap/sap-connector-reference-43.adoc [hunk 3]
```diff
@@ -0,0 +1,802 @@
...
+* SAP:RETRY_EXHAUSTED 
+* SAP:CONNECTIVITY 
+
+
+[[createIdoc]]
+=== Create IDoc
+`<sap:create-idoc>`
+
+
+Retrieves a IDocument based on its key.
```
Comment (george-hoffer) [inline modules/ROOT/pages/sap/sap-connector-reference-43.adoc:207]: GLOBAL: From a quick we search, it appears that IDocument is an incorrect usage - should either be IDoc or intermediate Document — see https://en.wikipedia.org/wiki/IDoc

**Review by george-hoffer — APPROVED** [review]

File: modules/ROOT/pages/sap/sap-connector.adoc [hunk 4]
```diff
@@ -4,23 +4,22 @@ include::_attributes.adoc[]
...
 :imagesdir: ../../assets/images/
 
-Support Category: Premium
+Support Category: Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
 
-SAP Connector v4.3
+SAP Connector v5.0
 
-The Anypoint Connector for SAP enables the integration of data to and from external systems based on SAP NetWeaver.
+The Anypoint Connector for SAP (SAP Connector) enables the integration of data to and from external systems based on SAP NetWeaver.
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector.adoc]: GLOBAL, no "the" before names:
```suggestion
Anypoint Connector for SAP (SAP Connector) enables the integration of data to and from external systems based on SAP NetWeaver.
```

File: modules/ROOT/pages/sap/sap-connector.adoc [hunk 5]
```diff
@@ -4,23 +4,22 @@ include::_attributes.adoc[]
...
 
 The SAP Connector is not available in the Design Center feature of Anypoint Platform.
 
 This connector requires a separate license. Contact MuleSoft to evaluate or use this connector in production.
 
-Mule runtime supports SAP integration through the Anypoint Connector for SAP, an SAP-certified Java connector that leverages the SAP Java Connector (JCo) libraries, thus allowing Mule applications to:
+Mule runtime engine supports SAP integration through the Anypoint Connector for SAP, an SAP-certified Java connector that leverages the SAP Java Connector (JCo) libraries, which allows Mule applications to:
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector.adoc]:
```suggestion
Mule runtime engine supports SAP integration through Anypoint Connector for SAP, an SAP-certified Java connector that leverages the SAP Java Connector (JCo) libraries, which allows Mule applications to:
```

File: modules/ROOT/pages/sap/sap-connector.adoc [hunk 6]
```diff
@@ -4,23 +4,22 @@ include::_attributes.adoc[]
...
 
-* Execute BAPI functions over the RFC protocol, supporting the following types:
-** Synchronous RFC (sRFC)
-** Transactional RFC (tRFC)
-** Queued RFC (qRFC)
-* Act as a JCo Server to be called as a BAPI over sRFC, tRFC, and qRFC.
+* Execute BAPI functions over the following types of RFC protocol:
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector.adoc]: types of protocols/type of protocol:
```suggestion
* Execute BAPI functions over the following types of RFC protocols:
```

File: modules/ROOT/pages/sap/sap-connector.adoc [hunk 7]
```diff
@@ -4,23 +4,22 @@ include::_attributes.adoc[]
...
+** Synchronous Remote Function Call RFC (sRFC)
+** Asynchronous Remote Function Call RFC (aRFC)
+* Act as a JCo Server to be called as a BAPI over sRFC, aRFC.
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector.adoc]:
```suggestion
* Act as a JCo server to be called as a BAPI over sRFC and aRFC.
```

File: modules/ROOT/pages/sap/sap-connector.adoc [hunk 8]
```diff
@@ -36,74 +35,75 @@ and resources on external systems, such as Salesforce, SAP, ServiceNow, and Twit
 
 This document assumes you have:
 
-* A working knowledge of the SAP business context and in particular, the SAP R/3 Business Suite.
+* A working knowledge of the SAP business context and, in particular, the SAP R/3 Business Suite.
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector.adoc]: I would remove periods at the ends of these list fragments, moving extra sentences below the list item....

File: modules/ROOT/pages/sap/sap-connector.adoc [hunk 9]
```diff
@@ -36,74 +35,75 @@ and resources on external systems, such as Salesforce, SAP, ServiceNow, and Twit
...
 * A basic understanding of the SAP NetWeaver Platform from an administration point of view.
 * Knowledge of the ABAP language.
-* Login credentials to test your connection to your target resource.
-* The SAP connector uses the RFC protocol to connect to NetWeaver Application Servers (NWAS). http://www.cipherbsc.com/solutions/sap-erp-central-component-erp-ecc/[ECC] and https://help.sap.com/viewer/p/SAP_CUSTOMER_RELATIONSHIP_MANAGEMENT[CRM] run on top of NWAS, as other SAP solutions do, and hence any customer using the connector may access those systems.
+* A SAP ID to download the required SAP libraries.
+* SAP login credentials to test your connection to your target resource.
+* Use the RFC protocol to connect to NetWeaver Application Servers (NWAS). http://www.cipherbsc.com/solutions/sap-erp-central-component-erp-ecc/[ECC] and https://help.sap.com/viewer/p/SAP_CUSTOMER_RELATIONSHIP_MANAGEMENT[CRM] run on top of NWAS, as other SAP solutions do, so any customer using the connector can access those systems.
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector.adoc]: Not grammatically in sync with the intro fragment (...you have:)
```suggestion
* Use of the RFC protocol to connect to NetWeaver Application Servers (NWAS). http://www.cipherbsc.com/solutions/sap-erp-central-component-erp-ecc/[ECC] and https://help.sap.com/viewer/p/SAP_CUSTOMER_RELATIONSHIP_MANAGEMENT[CRM] run on top of NWAS, as other SAP solutions do, so any customer using the connector can access those systems.
```

File: modules/ROOT/pages/sap/sap-connector.adoc [hunk 10]
```diff
@@ -36,74 +35,75 @@ and resources on external systems, such as Salesforce, SAP, ServiceNow, and Twit
...
-for IDOC, JCO, and JCO Native. The library formats are usable by
-Studio 7 directly without conversion. You can download them to
-a folder on your computer. You must have a SAP ID to download the libraries.
+for IDoc, JCO, and JCO Native to a folder on your computer. +
+The library formats are usable by Studio 7 directly without conversion. 
 
 == Audience
 
-* Starting user -- Configure your Mule application in 
+* Starting user -- Configure your Mule app in
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector.adoc]: GLOBAL CHANGE TO TEMPLATE TOO
```suggestion
* Starting user
Configure your Mule app in 
```

File: modules/ROOT/pages/sap/sap-connector.adoc [hunk 11]
```diff
@@ -36,74 +35,75 @@ and resources on external systems, such as Salesforce, SAP, ServiceNow, and Twit
...
 
 == Audience
 
-* Starting user -- Configure your Mule application in 
+* Starting user -- Configure your Mule app in 
 xref:sap/sap-connector-studio.adoc[Anypoint Studio]. The xref:sap/sap-connector-config-topics.adoc[Additional Configuration Information] topic helps you extend your understanding of the SAP Connector.
-* Power user - Read xref:sap/sap-connector-xml-maven.adoc[XML and Maven Support], 
-the xref:sap/sap-connector-config-topics.adoc[Additional Configuration Information] topic, 
-and xref:sap/sap-connector-examples.adoc[Examples].
+* Power user - Read the xref:sap/sap-connector-xml-maven.adoc[XML and Maven Support],
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector.adoc]:
```suggestion
* Power user
Read the xref:sap/sap-connector-xml-maven.adoc[XML and Maven Support], 
```

File: modules/ROOT/pages/sap/sap-connector.adoc [hunk 12]
```diff
@@ -36,74 +35,75 @@ and resources on external systems, such as Salesforce, SAP, ServiceNow, and Twit
...
-the xref:sap/sap-connector-config-topics.adoc[Additional Configuration Information] topic, 
-and xref:sap/sap-connector-examples.adoc[Examples].
+* Power user - Read the xref:sap/sap-connector-xml-maven.adoc[XML and Maven Support], 
+xref:sap/sap-connector-config-topics.adoc[Additional Configuration Information], 
+and xref:sap/sap-connector-examples.adoc[Examples] topics.
 
 == SAP Terminology
 
-ABAP (advanced business application programming)::
+ABAP (advanced business application programming) +
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector.adoc]: init cap full spelling

File: modules/ROOT/pages/sap/sap-connector.adoc [hunk 13]
```diff
@@ -36,74 +35,75 @@ and resources on external systems, such as Salesforce, SAP, ServiceNow, and Twit
...
 SAP proprietary programming language and part of the NetWeaver platform for building business applications.
 
-ALE (application link enabling)::
+ALE (application link enabling) +
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector.adoc]: init. cap.

File: modules/ROOT/pages/sap/sap-connector.adoc [hunk 14]
```diff
@@ -36,74 +35,75 @@ and resources on external systems, such as Salesforce, SAP, ServiceNow, and Twit
...
 Sets up distributed applications and integrated installation of SAP systems. This involves business-driven message exchange using consistent data across loosely linked SAP apps. Apps are integrated through synchronous and asynchronous communication, rather than by use of a central database.
 
-BAPI (business application programming interface)::
+BAPI (business application programming interface) +
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector.adoc]: ditto

File: modules/ROOT/pages/sap/sap-connector.adoc [hunk 15]
```diff
@@ -36,74 +35,75 @@ and resources on external systems, such as Salesforce, SAP, ServiceNow, and Twit
...
 ** Enable remotely.
 ** Run to completion with or without user interaction.
 ** Handle errors.
 
-FM (function module)::
+FM (function module) +
 Procedures defined in the ABAP language of SAP that allow the encapsulation and reuse of global functions in SAP.
 
-IDoc (Intermediate Document)::
+IDoc (Intermediate Document) +
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector.adoc]: lowercase

File: modules/ROOT/pages/sap/sap-connector.adoc [hunk 16]
```diff
@@ -36,74 +35,75 @@ and resources on external systems, such as Salesforce, SAP, ServiceNow, and Twit
...
 Standard SAP format for electronic data interchange between SAP systems. Different message types (such as delivery notes or purchase orders) generally correspond to different special formats, known as IDoc types. Multiple message types with related content can, however, be assigned to a single IDoc type.
 
-Java API::
+Java API +
 Handles dynamic metadata lookup and caching. The Java API implements the `JCO.Function` container used for parameters and/or tables for the SAP BAPI function module in Java. Java apps are built on top of the Java API.
 
-JNI (Java native interface)::
-Originally, SAP created libraries in C language to allow direct RFC calls to SAP, to manipulate with data. JCo wraps C libraries in Java to provide platform-native access into the SAP system. RFC middleware uses RFC library through the JNI Layer.
+JNI (Java native interface) +
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector.adoc]: init. cap.
lol...what a pain, eh?

File: modules/ROOT/pages/sap/sap-connector.adoc [hunk 17]
```diff
@@ -36,74 +35,75 @@ and resources on external systems, such as Salesforce, SAP, ServiceNow, and Twit
...
+Originally, SAP created libraries in C language to allow direct RFC calls to SAP, to manipulate with data. JCo wraps C libraries in Java to provide platform-native access into the SAP system. RFC middleware uses the RFC library through the JNI Layer.
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector.adoc:92]: I don't understand how ", to manipulate with data" works with the rest of the sentence? To manipulate what?

**Review by kbostrom — APPROVED** [review]
Done!

**Pull request merged**